src/ansys/dpf/core/operators/utility/merge_fields_containers.py

```diff
@@ -30,6 +30,14 @@ class merge_fields_containers(Operator):
     fields_containers2 : FieldsContainer
         A vector of fields containers to merge or
         fields containers from pin 0 to ...
+    should_merge_names_selection : bool, optional
```

Comment:
@MichaelNale could you fix the naming here and in the rest of these changes? We usually talk about `named selections` and not `names selections`